sdk/golang/ccnp/quote/proto/quote-server.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package quoteserver;
+option go_package = "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto/quoteServer";
+
+message HealthCheckRequest {
+  string service = 1;
+}
+
+message HealthCheckResponse {
+  enum ServingStatus {
+    UNKNOWN = 0;
+    SERVING = 1;
+    NOT_SERVING = 2;
+    SERVICE_UNKNOWN = 3;
```

Comment:
Maybe we need only define one of them between "UNKNOWN" and "SERVICE_UNKNOWN"?

Reply:
see: https://github.com/grpc/grpc/blob/master/doc/health-checking.md#service-definition